pkg/asset/installconfig/ovirt/cluster.go

```diff
@@ -15,22 +16,30 @@ func askCluster(c *ovirtsdk4.Connection, p *ovirt.Platform) (string, error) {
 	var clusterByNames = make(map[string]*ovirtsdk4.Cluster)
 	var clusterNames []string
 	systemService := c.SystemService()
-	response, err := systemService.ClustersService().List().Send()
+
+	dcResp, err := datacentersAvailable(c, "")
 	if err != nil {
-		return "", err
-	}
-	clusters, ok := response.Clusters()
-	if !ok {
-		return "", fmt.Errorf("there are no available clusters")
+		return "Failed to search Cluster available in the DataCenters!", err
```

Comment:
Return empty string

Reply:
Return empty string and err. When returning error it is better to not also return result

Reply:
@dougsland the return should look like `return "", errors.Wrap(err, "failed to list clusters")`